src/test/scala/com/danielasfregola/twitter4s/http/clients/streaming/statuses/TwitterStatusClientFS2Spec.scala

```diff
@@ -0,0 +1,46 @@
+// package com.danielasfregola.twitter4s.http.clients.streaming.statuses
+
+// import akka.http.scaladsl.model.HttpMethods
+// import com.danielasfregola.twitter4s.helpers.ClientSpec
+// import com.danielasfregola.twitter4s.entities.streaming.StreamingMessage
+
+// import scala.concurrent.ExecutionContext.Implicits.global
+
+// import cats.effect._
+// import fs2.Sink
+// import fs2.async.mutable.Queue
+
+// class TwitterStatusClientFS2Spec extends ClientSpec {
```

Comment:
[Note to everybody] I feel very nervous in merging without tests. We should find a way to test it before we can merge it